src/ConfigProvider.php

```diff
@@ -0,0 +1,365 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Laminas\View;
+
+use Laminas\ServiceManager\ConfigInterface;
+use Laminas\ServiceManager\Factory\InvokableFactory;
+use Laminas\ServiceManager\ServiceManager;
+
+use function array_merge;
+
+/**
+ * @psalm-import-type FactoriesConfigurationType from ConfigInterface
+ * @psalm-import-type ServiceManagerConfiguration from ServiceManager
+ * @psalm-type ViewHelperConfigurationType = array{
+ *     view_helpers?: ServiceManagerConfiguration,
+ *     view_helper_config?: array<string, array<array-key, mixed>
+ * }
+ * @psalm-suppress DeprecatedClass
+ */
+final class ConfigProvider
+{
+    /** @return ViewHelperConfigurationType */
+    public function __invoke(): array
+    {
+        return [
+            'view_helpers'       => self::viewHelperDependencyConfiguration(),
+            'view_helper_config' => [],
+        ];
+    }
+
+    /**
+     * @psalm-suppress DeprecatedClass
+     * @return ServiceManagerConfiguration
+     */
+    public static function viewHelperDependencyConfiguration(): array
```

Comment:
Please change this to non-static to stay in sync with almost every other `ConfigProvider` in the `laminas` org:

- https://github.com/laminas/laminas-diactoros/blob/d3c53c5e62f4fc9f3db95e952ece57a39c90b8e8/src/ConfigProvider.php#L32
- https://github.com/laminas/laminas-router/blob/44759e71620030c93d99e40b394fe9fff8f0beda/src/ConfigProvider.php#L37
- https://github.com/laminas/laminas-cache/blob/e2c9fef1f149655816035cc17d3b0f45e9e80a44/src/ConfigProvider.php#L37
- https://github.com/laminas/laminas-filter/blob/e8376ba0417d85028da0a61f7db50563baea8d22/src/ConfigProvider.php#L26
- [...]